node.js/tests/utils.js

```diff
@@ -0,0 +1,24 @@
+// Set Ingestion Key:
+exports.setEnv = (key, value) => {
+    process.env[key] = value;
+};
+
+// Set Test Data Information:
+exports.setData = (options) => {
```

Comment:
I personally prefer the ES6 way of exporting, `export function setData(options) {`
Also, `setData` is not what this does, it's returning sample test data. So maybe call it `getTestData`.
Also, I think it's better (and safer) if you spell out the specific parameters instead of wrapping it in `options`, `logGroup`, `logStream`, and `logMessage`

Reply:
Done!